service/vc/test_exports.go

```diff
@@ -276,11 +276,24 @@ func (env *DatabaseTestEnv) populateData( //nolint:revive
 	}))
 
 	for nsID, writes := range nsToWrites {
-		require.NoError(t, env.DB.retry.ExecuteSQL(t.Context(), env.DB.pool, fmt.Sprintf(`
-			INSERT INTO %s (key, value, version)
+		if writes.empty() {
+			continue
+		}
+
+		require.NotNil(t, writes.keys)
+		require.NotNil(t, writes.values)
+		require.NotNil(t, writes.versions)
+		require.Len(t, writes.values, len(writes.keys))
+		require.Len(t, writes.versions, len(writes.keys))
```

Comment:
I see this code at multiple places. What was the observation or need to introduce this check given that these helpers are used by the test code only? Was this introduced for debugging?